src/routes/object/copyObject.ts

```diff
@@ -29,9 +29,13 @@ const copyRequestBodySchema = {
 const successResponseSchema = {
   type: 'object',
   properties: {
+    Id: {
+      type: 'string',
+      examples: ['2eb16359-ecd4-4070-8eb1-8408baa42493'],
+    },
     Key: { type: 'string', examples: ['folder/destination.png'] },
   },
-  required: ['Key'],
+  required: ['Key', 'Id'],
```

Comment:
Let's change it to `id`. `Key` is the only place we use capital case - we should change that to `key` in the next breaking release